schemas/gwiu_writer_exit.fbs

```diff
@@ -0,0 +1,11 @@
+// File writer exit command
+//
+// Requests the file writer to exit
```

Comment:
Not sure why we need this. Starting the file writer will need to work through a different path. Why not the stopping as well?

Reply:
Good, that's also my opinion. I didn't want to drop functionality before discussing it with others though. I'll close the DM-1740 ticket which I made for that, and remove this schema.